java/src/main/java/io/opentracing/grpc/ActiveSpanSource.java

```diff
@@ -0,0 +1,38 @@
+package io.opentracing.contrib.grpc;
+
+import io.opentracing.Span;
+import io.opentracing.contrib.grpc.OpenTracingContextKey;
+
+/**
+ * An interface that defines how to get the current active span
+ */
+public interface ActiveSpanSource {
```

Comment:
(otherwise the users would have to manually put the current span into the grpc Context which might be annoying)